README.md

```diff
@@ -46,6 +46,49 @@ We take privacy seriously, and do not perform any automated error collection. In
 Thank you kindly!
 ```
 
+The error dump file generated by `human-panic` contains the following fields.
+
+```norun
+name = 'single-panic-test'
+operating_system = 'unix:Unknown'
+crate_version = '0.1.0'
+explanation = '''
+Cause: OMG EVERYTHING IS ON FIRE!!!. Panic occurred in file 'tests/single-panic/src/main.rs' at line 8
+'''
+method = 'Panic'
+backtrace = '''
+stack backtrace:
+   0:     0x55fa0ed4c1b4 - backtrace::backtrace::libunwind::trace::h69e50feca54bfb84
+                        at /home/spacekookie/.cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.6/src/backtrace/libunwind.rs:53
```

Comment:
can't reproduce this output locally, it says `/Users/pascal/.cargo/…` on my machine :trollface:

Reply:
Oh Pascal.... :trollface: